index.html

```diff
@@ -129,10 +129,76 @@ <h3 class="mb-4 section-subtitle text-bold text-proxima">What are my accomplishm
       </div>
       
     </section>
+
+    <section id="testimonials" class="pt-5 pb-5">
+      <div class="container pt-4 pb-4">
+        <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials</h2>
+
+        <div id="carouselExampleIndicators" class="carousel slide mb-5" data-ride="carousel">
+          <ol class="carousel-indicators">
+            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active">
+              <img
+                class="d-block img-fluid rounded-circle"
+                src="images/person-1.jpg"
+                srcset="images/person-1@2x.jpg 2x"
+                alt="Person 1"
+              />
+            </li>
+            <li data-target="#carouselExampleIndicators" data-slide-to="1">
```

Comment:
Use specific to this project name for the carousel. `carouselExampleIndicators ` doesn't make any sense.
I recommend:
```
data-target="#testimonials-carousel"
```